### ISSUE TITLE
Any new on the progress ?

### ISSUE THREAD

**Issue by Zbychus** [Nymria/TerrainGeneratorAndServer#7]
I am very much interested in generating QMT from the DEM source. I see several initiatives, however no result yet. Can you specify when you plan to have beta version or something we could give a try ?

regards

ZS

**Comment by MuhammedKalkan** [comment]
Hello,

Thank you for your interest in this work. I had to take some time off the project. Now i am actively working on it.
I have been testing several mesh simplification algorithms and trying to make it work  just like cesium server. I have made some progress. However results are not yet satisfying. 

I have declared here (https://github.com/Nymria/Cesium-Terrain-Generator-And-Server/issues/2) that whole algorithm will change in some future. And now it is time for that.

When this is done , it will provide generated meshes just like cesium server and will make generation faster. Also data merge offsets will be gone and hopefully seamless.

Keep an eye over here since i am trying to publish a new terrain set. 

Best regards
